metricbeat/module/mysql/status/data.go

```diff
@@ -125,6 +125,36 @@ var (
 				},
 				"write_requests": c.Int("Innodb_buffer_pool_write_requests"),
 			},
+			"rows": s.Object{
```

Comment:
Maybe we need to add some Unit tests in the future for this module so as to secure this schema won't break?
Sth similar to https://github.com/elastic/beats/blob/b5220465832b305f1ed3ae12838c4dd4fa345314/metricbeat/module/nats/connections/connections_test.go#L32

Reply:
Yeap, a full set of tests would be helpful and not just the ones that check the parent key. I have created an issue for it https://github.com/elastic/beats/issues/19880